content/src/fieldConfig/tax-clearance-certificate-shared.json

```diff
@@ -4,6 +4,11 @@
     "backButtonText": "Back",
     "stepperOneLabel": "Requirements",
     "stepperTwoLabel": "Check Eligibility",
-    "stepperThreeLabel": "Review"
+    "stepperThreeLabel": "Review",
+    "preHeaderErrorText": "Missing or invalid information.",
+    "singularErrorText": "Review the following field:",
+    "pluralErrorText": "Review the following fields:",
+    "requestingAgencyErrorText": "Select a Requesting Agency",
+    "businessNameErrorText": "Enter a Business Name"
```

Comment:
dust - should `businessNameErrorText` live with the business name field in the `profile.json`? There could be other cases where the business name input is required.

Reply:
I noticed validationText it provided via props, I don't expect to see this content changing when used across different content fields. Could this be a ticket that is immediately picked up?

Reply:
If we move this to `profile.json` and use globally, that implies we'll want the same error text everywhere. I'm also doubtful that we're going to need to change this, but I think this would be a new pattern for us.